[quant] quantized reflection_pad1d

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#37452 [quant] quantized reflection_pad1d**

Differential Revision: [D21286659](https://our.internmc.facebook.com/intern/diff/D21286659)

Commit:
[quant] quantized reflection_pad1d

raghuramank100 commented on test/quantization/test_quantized_op.py:
```diff
@@ -2524,6 +2524,30 @@ def test_qconv3d_unpack(
+           channels=st.integers(1, 64),
+           width=st.integers(16, 128),
+           qtype=st.sampled_from(hu._ALL_QINT_TYPES))
+    def test_reflection_pad1d(self, batch_size, channels, width, qtype):
```
Is this also tested for QNNPACK? Do you need to explicitly loop over qengines?

z-a-f replied:
Does it need qnnpack testing? It was not implemented in qnnpack, neither is it implemented in fbgemm. The implementation is purely in memory manipulation, and should be qengine agnostic

raghuramank100 reviewed (approved):
Question on qnnpack coverage.

Commit:
Update on "[quant] quantized reflection_pad1d"


Differential Revision: [D21286659](https://our.internmc.facebook.com/intern/diff/D21286659)

Commit:
Update on "[quant] quantized reflection_pad1d"


Differential Revision: [D21286659](https://our.internmc.facebook.com/intern/diff/D21286659)

Commit:
Update on "[quant] quantized reflection_pad1d"


Differential Revision: [D21286659](https://our.internmc.facebook.com/intern/diff/D21286659)
